MANUAL_QA.md

```
@@ -279,7 +287,105 @@ test-team-1/general/testuser > Line 1 Line 2 Line 3
+**実行手順**:
+1. 設定ファイルに無効な正規表現を追加：
+   ```json
+   "keywords": [
+     "/[invalid/i",
+     "/valid/i"
+   ]
+   ```
+2. アプリケーションを起動
+
+**期待結果**:
+- STDERRに無効な正規表現のエラーメッセージが表示される
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)